api/schemas/answer.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+import sqlmodel
+from sqlmodel import SQLModel, Field
+
+
+class Answer(SQLModel,table=True):
+    id: int | None = Field(primary_key=True, default=None)
+    exam_id: int = Field(default=None)  # TODO:: FK?
```

Comment:
```suggestion
    exam_id: str
```

No foreign key I think, as there's no exam item per se to store in the DB (it all comes from the YAML conf)

Reply:
Noted